packages/upswyng-web/src/components/__tests__/SearchInput.test.tsx

```diff
@@ -3,7 +3,8 @@ import React from "react";
 import SearchInput from "../SearchInput";
 
 jest.mock("react-router/Redirect", () => "Redirect");
-jest.mock("@material-ui/core/TextField", () => "TextField");
+jest.mock("@material-ui/core/Input", () => "Input");
+jest.mock("@material-ui/styles/makeStyles", () => () => () => ({}));
```

Comment:
fun

Reply:
no 😞  I'm going to need to figure out how to manually mock this at a global level so that this doesn't have to happen in each component where `makeStyles` is used.